lib/IHP/static/helpers.js

```diff
@@ -49,7 +49,8 @@ function initDelete() {
             event.preventDefault();
 
             if (!event.target.classList.contains('js-delete-no-confirm')) {
-                if (!confirm('Are you sure you want to delete this?')) {
+                let txt = event.target.dataset.confirmtext;
```

Comment:
Can we use `var` here instead. I'd also shorten the name to just `data-confirm` instead of `data-confirmtext`.

```suggestion
                var confirmText = event.target.dataset.confirm || 'Are you sure you want to delete this?';
```